database/openshift.deploy.yml

```diff
@@ -28,10 +28,6 @@ parameters:
     value: "50m"
   - name: MEMORY_REQUEST
     value: "250Mi"
-  - name: CPU_LIMIT
```

Comment:
We'll need to cleanup this folder after we've confirmed nothing is in use (I don't think it is) so don't worry about this file